package.json

```diff
@@ -2,6 +2,17 @@
   "name": "create-html5-boilerplate",
   "version": "0.0.6",
   "description": "",
+  "keywords": [],
+  "homepage": "https://html5boilerplate.com/",
+  "bugs": {
+    "url": "https://github.com/h5bp/create-html5-boilerplate/issues"
+  },
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/h5bp/create-html5-boilerplate.git"
+  },
+  "license": "ISC",
```

Comment:
We've generally done MIT in the rest of the HTML5-Boilerplate projects

Reply:
yep, I think the initial `package.json` committed was just using the npm init defaults.

Reply:
Yep, I used default config